blogs/dns_tracking/dns_tracking.md

```diff
@@ -0,0 +1,138 @@
+# Network Observability Per Flow DNS tracking
+
+![logo](./images/dns_tracking_logo.png)
+
+By: Julien Pinsonneau, Mehul Modi and Mohamed S. Mahmoud
+
+In today's interconnected digital landscape, Domain Name System (DNS) tracking
+plays a crucial role in networking and security. DNS resolution is a fundamental
+process that translates human-readable domain names into IP addresses, enabling
+communication between devices and servers. It can also be a security vulnerability
+and a tool for firewalling and blocking access to content, depending on how it is
+configured and manipulated. For operations excellence, DNS
+resolution benefits from monitoring and analysis, which can be achieved through
+innovative technologies like eBPF (extended Berkeley Packet Filter). In this
+blog post, we'll delve into the world of DNS tracking using eBPF tracepoint
+hooks, exploring how this powerful combination can be used for various purposes,
+including network monitoring and security enhancement.
+
+## Understanding DNS Resolution
+
+Before diving into the specifics of eBPF tracepoint hooks, let's briefly recap
+how DNS resolution works. In a Kubernetes architecture, DNS (Domain Name System)
```

Comment:
```suggestion
how DNS resolution works. In a Kubernetes architecture, DNS
```

Reply:
Since you already defined this acronym earlier, I don't think you need to define it again. In product docs, we typically only define for the first mention on the page then the rest of the mentions are acronym.

Reply:
thanks ! https://github.com/netobserv/documents/pull/53/commits/29c84753d09a32904baa80885f7b9ab50896a4ee